<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>390</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_registrar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Registra e salva os dados inseridos</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
padding: 5px 3px;
background-color: #4CAF50;
border-radius: 6px;
}


QPushButton:hover{
background-color: green;
}



QPushButton:pressed{
  background-color: #3e8e41;
}

</string>
    </property>
    <property name="text">
     <string>registrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>código remédio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>numero da nota fiscal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>metodo de pagamento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_codigo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font-size: 12px;
text-align: center;
border-radius: 1px;
background-color: rgb(230, 230, 230);
border: 1px solid  rgb(12, 12, 12);
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_qtd">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font-size: 12px;
text-align: center;
border-radius: 1px;
background-color: rgb(230, 230, 230);
border: 1px solid  rgb(12, 12, 12);
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nota">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font-size: 12px;
text-align: center;
border-radius: 1px;
background-color: rgb(230, 230, 230);
border: 1px solid  rgb(12, 12, 12);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_aprazo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>a prazo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_avista">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>a vista</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrar vendas</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_editavel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_ver_registro">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Ir para a tela de registros</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
padding: 5px 3px;
background-color: #4CAF50;
border-radius: 6px;
}


QPushButton:hover{
background-color: green;
}



QPushButton:pressed{
  background-color: #3e8e41;
}

</string>
    </property>
    <property name="text">
     <string>ver registro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_voltar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Voltar a tela de dashboard</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
padding: 5px 3px;
background-color: #4CAF50;
border-radius: 15px;
}


QPushButton:hover{
background-color: green;
}



QPushButton:pressed{
  background-color: #3e8e41;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/sair.png</normaloff>icons/sair.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
